### PR TITLE
D3ASIM-1995: Added restrictions on max amount of posted energy for bi…

### PR DESCRIPTION
**Pull request by spyrostz**
…ds and offers for loads and pvs respectively.

### CODE REVIEW

File: src/d3a/models/strategy/external_strategies/load.py [hunk 1]
```diff
@@ -87,6 +87,15 @@ def _bid(self, payload):
             arguments = json.loads(payload["data"])
             assert set(arguments.keys()) == {'price', 'energy'}
             arguments['buyer_origin'] = self.device.name
+            posted_energy = (arguments['price'] + self.posted_bid_energy(self.market.id)) * 1000.0
```
Comment (hannesdiedrich) [inline src/d3a/models/strategy/external_strategies/load.py]: I think you meant `arguments['energy']` here, right?

File: src/d3a/models/strategy/external_strategies/pv.py [hunk 2]
```diff
@@ -92,6 +92,16 @@ def _offer(self, payload):
             assert set(arguments.keys()) == {'price', 'energy'}
             arguments['seller'] = self.device.name
             arguments['seller_origin'] = self.device.name
+
+            posted_energy = (arguments['price'] + self.posted_offer_energy(self.market.id))
```
Comment (hannesdiedrich) [inline src/d3a/models/strategy/external_strategies/pv.py]: Same as above.

File: src/d3a/models/strategy/external_strategies/load.py (hunk 1 above)
Comment (spyrostz) [reply src/d3a/models/strategy/external_strategies/load.py]: Should be fixed now, thanks!

File: src/d3a/models/strategy/external_strategies/pv.py (hunk 2 above)
Comment (spyrostz) [reply src/d3a/models/strategy/external_strategies/pv.py]: Fixed, thanks

**Pull request merged**